source/expressions.tex

```diff
@@ -4166,10 +4166,10 @@
 The unary \tcode{*} operator performs \defn{indirection}:
 \indextext{dereferencing|see{indirection}}%
 the expression to which it is applied shall be a pointer to an object
-type, or a pointer to a function type and the result is an lvalue
+type, or a pointer to a function type and the resulting expression is an lvalue
 referring to the object or function to which the expression points. If
```

Comment:
If some renovation is desired here, replacing "expression" with "operand" would probably help. Also, integrating the "pointer to T" phrasing with the preceding "shall be a pointer to object type" rule would probably help.

Maybe something like "The unary * operator performs indirection. Its operand shall be a prvalue of type "pointer to T", where T is an object or function type. The lvalue result of type T refers to the object or function to which the operand points."